### ISSUE TITLE
Change /candidates/totals to default to election_full=true

### ISSUE THREAD

**Issue by jason-upchurch** [fecgov/openFEC#3686]
There is `active_candidates` flag being used on Raising and Spending bar charts. This flag can effect the overall counts.

One example mismatch:

Counts for candidates as of 4/10/2019 = 664
- https://api.open.fec.gov/v1/candidates/?election_full=false&api_key=DEMO_KEY&sort_null_only=false&sort_nulls_last=false&election_year=2020&office=P&page=1&per_page=20&sort_hide_null=false

Counts for candidates as of 4/10/2019 = 510
- https://api.open.fec.gov/v1/candidates/totals/?election_full=false&api_key=DEMO_KEY&sort_null_only=false&sort_nulls_last=false&election_year=2020&office=P&page=1&per_page=20&sort_hide_null=false

**Findings**
After discussing with @fecjjeng, `election_full` isn't meaningful for house candidates in /candidates/totals and we should change the default in the API to election_full is True

**Completion criteria**
- [x] Make the change to /candidates/totals to make `election_full=true`
- [x] Make sure this doesn't change the behavior for house/senate/presidential candidate data tables for full cycle and 2-year breakdowns
- [x] Check the data, including special election candidates

**Comment by qqss88** [comment]
Here is the sql query for the candidate endpoint request:
SELECT ofec_candidate_detail_mv.idx AS ofec_candidate_detail_mv_idx, ofec_candidate_detail_mv.name AS ofec_candidate_detail_mv_name, ofec_candidate_detail_mv.office AS ofec_candidate_detail_mv_office, ofec_candidate_detail_mv.office_full AS ofec_candidate_detail_mv_office_full, ofec_candidate_detail_mv.party AS ofec_candidate_detail_mv_party, ofec_candidate_detail_mv.party_full AS ofec_candidate_detail_mv_party_full, ofec_candidate_detail_mv.state AS ofec_candidate_detail_mv_state, ofec_candidate_detail_mv.district AS ofec_candidate_detail_mv_district, ofec_candidate_detail_mv.district_number AS ofec_candidate_detail_mv_district_number, ofec_candidate_detail_mv.election_districts AS ofec_candidate_detail_mv_election_districts, ofec_candidate_detail_mv.election_years AS ofec_candidate_detail_mv_election_years, ofec_candidate_detail_mv.cycles AS ofec_candidate_detail_mv_cycles, ofec_candidate_detail_mv.candidate_status AS ofec_candidate_detail_mv_candidate_status, ofec_candidate_detail_mv.incumbent_challenge AS ofec_candidate_detail_mv_incumbent_challenge, ofec_candidate_detail_mv.incumbent_challenge_full AS ofec_candidate_detail_mv_incumbent_challenge_full, ofec_candidate_detail_mv.load_date AS ofec_candidate_detail_mv_load_date, ofec_candidate_detail_mv.first_file_date AS ofec_candidate_detail_mv_first_file_date, ofec_candidate_detail_mv.last_file_date AS ofec_candidate_detail_mv_last_file_date, ofec_candidate_detail_mv.last_f2_date AS ofec_candidate_detail_mv_last_f2_date, ofec_candidate_detail_mv.candidate_id AS ofec_candidate_detail_mv_candidate_id, ofec_candidate_detail_mv.active_through AS ofec_candidate_detail_mv_active_through, ofec_candidate_flag_mv_1.candidate_id AS ofec_candidate_flag_mv_1_candidate_id, ofec_candidate_flag_mv_1.federal_funds_flag AS ofec_candidate_flag_mv_1_federal_funds_flag, ofec_candidate_flag_mv_1.has_raised_funds AS ofec_candidate_flag_mv_1_has_raised_funds
FROM ofec_candidate_detail_mv LEFT OUTER JOIN ofec_candidate_flag_mv AS ofec_candidate_flag_mv_1 ON ofec_candidate_flag_mv_1.candidate_id = ofec_candidate_detail_mv.candidate_id
WHERE ofec_candidate_detail_mv.office IN ('P') AND ofec_candidate_detail_mv.election_years && Array[2020]
ORDER BY ofec_candidate_detail_mv.name ASC;

Current query result count is 665;

**Comment by qqss88** [comment]
the sql query for the candidate total api:
SELECT ofec_candidate_history_with_future_election_mv.idx AS ofec_candidate_history_with_future_election_mv_idx, ofec_candidate_history_with_future_election_mv.name AS ofec_candidate_history_with_future_election_mv_name, ofec_candidate_history_with_future_election_mv.office AS ofec_candidate_history_with_future_election_mv_office, ofec_candidate_history_with_future_election_mv.office_full AS ofec_candidate_history_with_future_election_mv_office_ful_1, ofec_candidate_history_with_future_election_mv.party AS ofec_candidate_history_with_future_election_mv_party, ofec_candidate_history_with_future_election_mv.party_full AS ofec_candidate_history_with_future_election_mv_party_full, ofec_candidate_history_with_future_election_mv.state AS ofec_candidate_history_with_future_election_mv_state, ofec_candidate_history_with_future_election_mv.district AS ofec_candidate_history_with_future_election_mv_district, ofec_candidate_history_with_future_election_mv.district_number AS ofec_candidate_history_with_future_election_mv_district_n_2, ofec_candidate_history_with_future_election_mv.election_districts AS ofec_candidate_history_with_future_election_mv_election_d_3, ofec_candidate_history_with_future_election_mv.election_years AS ofec_candidate_history_with_future_election_mv_election_y_4, ofec_candidate_history_with_future_election_mv.cycles AS ofec_candidate_history_with_future_election_mv_cycles, ofec_candidate_history_with_future_election_mv.candidate_status AS ofec_candidate_history_with_future_election_mv_candidate__5, ofec_candidate_history_with_future_election_mv.incumbent_challenge AS ofec_candidate_history_with_future_election_mv_incumbent__6, ofec_candidate_history_with_future_election_mv.incumbent_challenge_full AS ofec_candidate_history_with_future_election_mv_incumbent__7, ofec_candidate_history_with_future_election_mv.load_date AS ofec_candidate_history_with_future_election_mv_load_date, ofec_candidate_history_with_future_election_mv.first_file_date AS ofec_candidate_history_with_future_election_mv_first_file_8, ofec_candidate_history_with_future_election_mv.last_file_date AS ofec_candidate_history_with_future_election_mv_last_file__9, ofec_candidate_history_with_future_election_mv.last_f2_date AS ofec_candidate_history_with_future_election_mv_last_f2_da_a, ofec_candidate_history_with_future_election_mv.candidate_id AS ofec_candidate_history_with_future_election_mv_candidate__b, ofec_candidate_history_with_future_election_mv.two_year_period AS ofec_candidate_history_with_future_election_mv_two_year_p_c, ofec_candidate_history_with_future_election_mv.candidate_election_year AS ofec_candidate_history_with_future_election_mv_candidate__d, ofec_candidate_history_with_future_election_mv.address_city AS ofec_candidate_history_with_future_election_mv_address_ci_e, ofec_candidate_history_with_future_election_mv.address_state AS ofec_candidate_history_with_future_election_mv_address_st_f, ofec_candidate_history_with_future_election_mv.address_street_1 AS ofec_candidate_history_with_future_election_mv_address_st_10, ofec_candidate_history_with_future_election_mv.address_street_2 AS ofec_candidate_history_with_future_election_mv_address_st_11, ofec_candidate_history_with_future_election_mv.address_zip AS ofec_candidate_history_with_future_election_mv_address_zi_12, ofec_candidate_history_with_future_election_mv.candidate_inactive AS ofec_candidate_history_with_future_election_mv_candidate__13, ofec_candidate_history_with_future_election_mv.active_through AS ofec_candidate_history_with_future_election_mv_active_thr_14, ofec_candidate_totals_mv.candidate_id AS ofec_candidate_totals_mv_candidate_id, ofec_candidate_totals_mv.election_year AS ofec_candidate_totals_mv_election_year, ofec_candidate_totals_mv.cycle AS ofec_candidate_totals_mv_cycle, ofec_candidate_totals_mv.is_election AS ofec_candidate_totals_mv_is_election, ofec_candidate_totals_mv.receipts AS ofec_candidate_totals_mv_receipts, ofec_candidate_totals_mv.disbursements AS ofec_candidate_totals_mv_disbursements, ofec_candidate_totals_mv.cash_on_hand_end_period AS ofec_candidate_totals_mv_cash_on_hand_end_period, ofec_candidate_totals_mv.debts_owed_by_committee AS ofec_candidate_totals_mv_debts_owed_by_committee, ofec_candidate_totals_mv.coverage_start_date AS ofec_candidate_totals_mv_coverage_start_date, ofec_candidate_totals_mv.coverage_end_date AS ofec_candidate_totals_mv_coverage_end_date, ofec_candidate_totals_mv.federal_funds_flag AS ofec_candidate_totals_mv_federal_funds_flag, ofec_candidate_totals_mv.has_raised_funds AS ofec_candidate_totals_mv_has_raised_funds, ofec_candidate_totals_mv.party AS ofec_candidate_totals_mv_party, ofec_candidate_totals_mv.office AS ofec_candidate_totals_mv_office, ofec_candidate_totals_mv.candidate_inactive AS ofec_candidate_totals_mv_candidate_inactive
FROM ofec_candidate_history_with_future_election_mv JOIN ofec_candidate_totals_mv ON ofec_candidate_history_with_future_election_mv.candidate_id = ofec_candidate_totals_mv.candidate_id AND ofec_candidate_history_with_future_election_mv.two_year_period = ofec_candidate_totals_mv.cycle JOIN ofec_candidate_detail_mv ON ofec_candidate_history_with_future_election_mv.candidate_id = ofec_candidate_detail_mv.candidate_id
WHERE ofec_candidate_totals_mv.election_year IN (2020) AND ofec_candidate_history_with_future_election_mv.office IN ('P') AND ofec_candidate_totals_mv.is_election = false;

current count result is 548

**Comment by qqss88** [comment]
candidate total query has candidate-id redundancy, like this one - P00000679, one for each cycle(2018, 2020):
{
"active_through": 2020,
"two_year_period": 2018,
"has_raised_funds": true,
"coverage_end_date": "2018-12-31T00:00:00",
"address_city": "STOCKTON",
"election_years": [
1980,
1984,
1988,
1992,
2000,
2004,
2008,
2012,
2016,
2020
],
"party_full": "NONE",
"first_file_date": "1979-04-06",
"load_date": "2018-02-17T09:16:20+00:00",
"address_zip": "95208",
"incumbent_challenge_full": null,
"office_full": "President",
"candidate_id": "P00000679",
"address_street_1": "PO BOX 9079",
"last_file_date": "2017-04-18",
"cycle": 2018,
"cycles": [
1980,
1982,
1984,
1986,
1988,
1990,
1992,
1994,
1996,
1998,
2000,
2002,
2004,
2006,
2008,
2010,
2012,
2014,
2016,
2018,
2020
],
"district_number": 0,
"candidate_status": "N",
"last_f2_date": "2017-04-18",
"federal_funds_flag": false,
"candidate_inactive": false,
"receipts": 160,
"state": "US",
"district": "00",
"is_election": false,
"debts_owed_by_committee": 3019,
"office": "P",
"election_year": 2020,
"cash_on_hand_end_period": 2,
"candidate_election_year": 2020,
"address_street_2": null,
"party": "NNE",
"name": "CARROLL, JERRY LEON",
"election_districts": [
"00",
"00",
"00",
"00",
"00",
"00",
"00",
"00",
"00",
"00"
],
"coverage_start_date": "2016-12-31T00:00:00",
"incumbent_challenge": null,
"address_state": "CA",
"disbursements": 160
},
{
"active_through": 2020,
"two_year_period": 2020,
"has_raised_funds": true,
"coverage_end_date": "2019-03-31T00:00:00",
"address_city": "STOCKTON",
"election_years": [
1980,
1984,
1988,
1992,
2000,
2004,
2008,
2012,
2016,
2020
],
"party_full": "NONE",
"first_file_date": "1979-04-06",
"load_date": "2019-01-28T21:05:11+00:00",
"address_zip": "95208",
"incumbent_challenge_full": "Challenger",
"office_full": "President",
"candidate_id": "P00000679",
"address_street_1": "PO BOX 9079",
"last_file_date": "2017-04-18",
"cycle": 2020,
"cycles": [
1980,
1982,
1984,
1986,
1988,
1990,
1992,
1994,
1996,
1998,
2000,
2002,
2004,
2006,
2008,
2010,
2012,
2014,
2016,
2018,
2020
],
"district_number": 0,
"candidate_status": "N",
"last_f2_date": "2017-04-18",
"federal_funds_flag": false,
"candidate_inactive": false,
"receipts": 20,
"state": "US",
"district": "00",
"is_election": false,
"debts_owed_by_committee": 3039,
"office": "P",
"election_year": 2020,
"cash_on_hand_end_period": 2,
"candidate_election_year": 2020,
"address_street_2": null,
"party": "NNE",
"name": "CARROLL, JERRY LEON",
"election_districts": [
"00",
"00",
"00",
"00",
"00",
"00",
"00",
"00",
"00",
"00"
],
"coverage_start_date": "2018-12-31T00:00:00",
"incumbent_challenge": "C",
"address_state": "CA",
"disbursements": 20
},

**Comment by qqss88** [comment]
unique candidate id list from candidate query: 
665

unique caidiate id list from candidate/total query:
457 (after removing redundancy from 548 returned items)

**Comment by qqss88** [comment]
curl 'https://api.open.fec.gov/v1/candidates/?api_key=5yyI90SU3Xb73TVlv4wrEhQxYcCwMWCywQiGdYbJ&sort_hide_null=false&sort_nulls_last=true&election_year=2020&office=P&sort=-first_file_date&per_page=30&page=1' -H 'Accept: application/json, text/javascript, */*; q=0.01' -H 'Referer: https://www.fec.gov/data/candidates/?election_year=2020&office=P' -H 'Origin: https://www.fec.gov' -H 'User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.103 Safari/537.36' --compressed

**Comment by qqss88** [comment]
when the election_year is set false, this query return 6 items corresponding to 6 historical cycles:
SELECT ofec_candidate_history_with_future_election_mv.idx AS ofec_candidate_history_with_future_election_mv_idx, ofec_candidate_history_with_future_election_mv.name AS ofec_candidate_history_with_future_election_mv_name, ofec_candidate_history_with_future_election_mv.office AS ofec_candidate_history_with_future_election_mv_office, ofec_candidate_history_with_future_election_mv.office_full AS ofec_candidate_history_with_future_election_mv_office_ful_1, ofec_candidate_history_with_future_election_mv.party AS ofec_candidate_history_with_future_election_mv_party, ofec_candidate_history_with_future_election_mv.party_full AS ofec_candidate_history_with_future_election_mv_party_full, ofec_candidate_history_with_future_election_mv.state AS ofec_candidate_history_with_future_election_mv_state, ofec_candidate_history_with_future_election_mv.district AS ofec_candidate_history_with_future_election_mv_district, ofec_candidate_history_with_future_election_mv.district_number AS ofec_candidate_history_with_future_election_mv_district_n_2, ofec_candidate_history_with_future_election_mv.election_districts AS ofec_candidate_history_with_future_election_mv_election_d_3, ofec_candidate_history_with_future_election_mv.election_years AS ofec_candidate_history_with_future_election_mv_election_y_4, ofec_candidate_history_with_future_election_mv.cycles AS ofec_candidate_history_with_future_election_mv_cycles, ofec_candidate_history_with_future_election_mv.candidate_status AS ofec_candidate_history_with_future_election_mv_candidate__5, ofec_candidate_history_with_future_election_mv.incumbent_challenge AS ofec_candidate_history_with_future_election_mv_incumbent__6, ofec_candidate_history_with_future_election_mv.incumbent_challenge_full AS ofec_candidate_history_with_future_election_mv_incumbent__7, ofec_candidate_history_with_future_election_mv.load_date AS ofec_candidate_history_with_future_election_mv_load_date, ofec_candidate_history_with_future_election_mv.first_file_date AS ofec_candidate_history_with_future_election_mv_first_file_8, ofec_candidate_history_with_future_election_mv.last_file_date AS ofec_candidate_history_with_future_election_mv_last_file__9, ofec_candidate_history_with_future_election_mv.last_f2_date AS ofec_candidate_history_with_future_election_mv_last_f2_da_a, ofec_candidate_history_with_future_election_mv.candidate_id AS ofec_candidate_history_with_future_election_mv_candidate__b, ofec_candidate_history_with_future_election_mv.two_year_period AS ofec_candidate_history_with_future_election_mv_two_year_p_c, ofec_candidate_history_with_future_election_mv.candidate_election_year AS ofec_candidate_history_with_future_election_mv_candidate__d, ofec_candidate_history_with_future_election_mv.address_city AS ofec_candidate_history_with_future_election_mv_address_ci_e, ofec_candidate_history_with_future_election_mv.address_state AS ofec_candidate_history_with_future_election_mv_address_st_f, ofec_candidate_history_with_future_election_mv.address_street_1 AS ofec_candidate_history_with_future_election_mv_address_st_10, ofec_candidate_history_with_future_election_mv.address_street_2 AS ofec_candidate_history_with_future_election_mv_address_st_11, ofec_candidate_history_with_future_election_mv.address_zip AS ofec_candidate_history_with_future_election_mv_address_zi_12, ofec_candidate_history_with_future_election_mv.candidate_inactive AS ofec_candidate_history_with_future_election_mv_candidate__13, ofec_candidate_history_with_future_election_mv.active_through AS ofec_candidate_history_with_future_election_mv_active_thr_14, ofec_candidate_totals_mv.candidate_id AS ofec_candidate_totals_mv_candidate_id, ofec_candidate_totals_mv.election_year AS ofec_candidate_totals_mv_election_year, ofec_candidate_totals_mv.cycle AS ofec_candidate_totals_mv_cycle, ofec_candidate_totals_mv.is_election AS ofec_candidate_totals_mv_is_election, ofec_candidate_totals_mv.receipts AS ofec_candidate_totals_mv_receipts, ofec_candidate_totals_mv.disbursements AS ofec_candidate_totals_mv_disbursements, ofec_candidate_totals_mv.cash_on_hand_end_period AS ofec_candidate_totals_mv_cash_on_hand_end_period, ofec_candidate_totals_mv.debts_owed_by_committee AS ofec_candidate_totals_mv_debts_owed_by_committee, ofec_candidate_totals_mv.coverage_start_date AS ofec_candidate_totals_mv_coverage_start_date, ofec_candidate_totals_mv.coverage_end_date AS ofec_candidate_totals_mv_coverage_end_date, ofec_candidate_totals_mv.federal_funds_flag AS ofec_candidate_totals_mv_federal_funds_flag, ofec_candidate_totals_mv.has_raised_funds AS ofec_candidate_totals_mv_has_raised_funds, ofec_candidate_totals_mv.party AS ofec_candidate_totals_mv_party, ofec_candidate_totals_mv.office AS ofec_candidate_totals_mv_office, ofec_candidate_totals_mv.candidate_inactive AS ofec_candidate_totals_mv_candidate_inactive
FROM ofec_candidate_history_with_future_election_mv JOIN ofec_candidate_totals_mv ON ofec_candidate_history_with_future_election_mv.candidate_id = ofec_candidate_totals_mv.candidate_id AND ofec_candidate_history_with_future_election_mv.two_year_period = ofec_candidate_totals_mv.cycle JOIN ofec_candidate_detail_mv ON ofec_candidate_history_with_future_election_mv.candidate_id = ofec_candidate_detail_mv.candidate_id
WHERE ofec_candidate_totals_mv.election_year IN (2020) AND ofec_candidate_history_with_future_election_mv.office IN ('P') AND ofec_candidate_totals_mv.is_election = false AND ofec_candidate_totals_mv.candidate_id = 'P80004492'; 


![image](https://user-images.githubusercontent.com/7330555/56389570-e2b77100-61f7-11e9-9348-b4697aad6aa4.png)

**Comment by qqss88** [comment]
3 MVs involved in query candidate/totals endpoint:
ofec_candidate_history_with_future_election_mv
ofec_candidate_totals_mv
ofec_candidate_detail_mv

**Comment by qqss88** [comment]
for candidate like 'P00010256' (cand-specific one), it only have one item existed in the related MV query with is_election = True:

SELECT ofec_candidate_history_with_future_election_mv.idx AS ofec_candidate_history_with_future_election_mv_idx, ofec_candidate_history_with_future_election_mv.name AS ofec_candidate_history_with_future_election_mv_name, ofec_candidate_history_with_future_election_mv.office AS ofec_candidate_history_with_future_election_mv_office, ofec_candidate_history_with_future_election_mv.office_full AS ofec_candidate_history_with_future_election_mv_office_ful_1, ofec_candidate_history_with_future_election_mv.party AS ofec_candidate_history_with_future_election_mv_party, ofec_candidate_history_with_future_election_mv.party_full AS ofec_candidate_history_with_future_election_mv_party_full, ofec_candidate_history_with_future_election_mv.state AS ofec_candidate_history_with_future_election_mv_state, ofec_candidate_history_with_future_election_mv.district AS ofec_candidate_history_with_future_election_mv_district, ofec_candidate_history_with_future_election_mv.district_number AS ofec_candidate_history_with_future_election_mv_district_n_2, ofec_candidate_history_with_future_election_mv.election_districts AS ofec_candidate_history_with_future_election_mv_election_d_3, ofec_candidate_history_with_future_election_mv.election_years AS ofec_candidate_history_with_future_election_mv_election_y_4, ofec_candidate_history_with_future_election_mv.cycles AS ofec_candidate_history_with_future_election_mv_cycles, ofec_candidate_history_with_future_election_mv.candidate_status AS ofec_candidate_history_with_future_election_mv_candidate__5, ofec_candidate_history_with_future_election_mv.incumbent_challenge AS ofec_candidate_history_with_future_election_mv_incumbent__6, ofec_candidate_history_with_future_election_mv.incumbent_challenge_full AS ofec_candidate_history_with_future_election_mv_incumbent__7, ofec_candidate_history_with_future_election_mv.load_date AS ofec_candidate_history_with_future_election_mv_load_date, ofec_candidate_history_with_future_election_mv.first_file_date AS ofec_candidate_history_with_future_election_mv_first_file_8, ofec_candidate_history_with_future_election_mv.last_file_date AS ofec_candidate_history_with_future_election_mv_last_file__9, ofec_candidate_history_with_future_election_mv.last_f2_date AS ofec_candidate_history_with_future_election_mv_last_f2_da_a, ofec_candidate_history_with_future_election_mv.candidate_id AS ofec_candidate_history_with_future_election_mv_candidate__b, ofec_candidate_history_with_future_election_mv.two_year_period AS ofec_candidate_history_with_future_election_mv_two_year_p_c, ofec_candidate_history_with_future_election_mv.candidate_election_year AS ofec_candidate_history_with_future_election_mv_candidate__d, ofec_candidate_history_with_future_election_mv.address_city AS ofec_candidate_history_with_future_election_mv_address_ci_e, ofec_candidate_history_with_future_election_mv.address_state AS ofec_candidate_history_with_future_election_mv_address_st_f, ofec_candidate_history_with_future_election_mv.address_street_1 AS ofec_candidate_history_with_future_election_mv_address_st_10, ofec_candidate_history_with_future_election_mv.address_street_2 AS ofec_candidate_history_with_future_election_mv_address_st_11, ofec_candidate_history_with_future_election_mv.address_zip AS ofec_candidate_history_with_future_election_mv_address_zi_12, ofec_candidate_history_with_future_election_mv.candidate_inactive AS ofec_candidate_history_with_future_election_mv_candidate__13, ofec_candidate_history_with_future_election_mv.active_through AS ofec_candidate_history_with_future_election_mv_active_thr_14, ofec_candidate_totals_mv.candidate_id AS ofec_candidate_totals_mv_candidate_id, ofec_candidate_totals_mv.election_year AS ofec_candidate_totals_mv_election_year, ofec_candidate_totals_mv.cycle AS ofec_candidate_totals_mv_cycle, ofec_candidate_totals_mv.is_election AS ofec_candidate_totals_mv_is_election, ofec_candidate_totals_mv.receipts AS ofec_candidate_totals_mv_receipts, ofec_candidate_totals_mv.disbursements AS ofec_candidate_totals_mv_disbursements, ofec_candidate_totals_mv.cash_on_hand_end_period AS ofec_candidate_totals_mv_cash_on_hand_end_period, ofec_candidate_totals_mv.debts_owed_by_committee AS ofec_candidate_totals_mv_debts_owed_by_committee, ofec_candidate_totals_mv.coverage_start_date AS ofec_candidate_totals_mv_coverage_start_date, ofec_candidate_totals_mv.coverage_end_date AS ofec_candidate_totals_mv_coverage_end_date, ofec_candidate_totals_mv.federal_funds_flag AS ofec_candidate_totals_mv_federal_funds_flag, ofec_candidate_totals_mv.has_raised_funds AS ofec_candidate_totals_mv_has_raised_funds, ofec_candidate_totals_mv.party AS ofec_candidate_totals_mv_party, ofec_candidate_totals_mv.office AS ofec_candidate_totals_mv_office, ofec_candidate_totals_mv.candidate_inactive AS ofec_candidate_totals_mv_candidate_inactive
FROM ofec_candidate_history_with_future_election_mv JOIN ofec_candidate_totals_mv ON ofec_candidate_history_with_future_election_mv.candidate_id = ofec_candidate_totals_mv.candidate_id AND ofec_candidate_history_with_future_election_mv.two_year_period = ofec_candidate_totals_mv.cycle JOIN ofec_candidate_detail_mv ON ofec_candidate_history_with_future_election_mv.candidate_id = ofec_candidate_detail_mv.candidate_id
WHERE ofec_candidate_totals_mv.election_year IN (2020) AND ofec_candidate_history_with_future_election_mv.office IN ('P') AND ofec_candidate_totals_mv.candidate_id = 'P00010256';

<img width="507" alt="image" src="https://user-images.githubusercontent.com/7330555/56471178-7322cb00-641d-11e9-9c65-79fe3bccd5b0.png">

**Comment by qqss88** [comment]
need to clarify the relationship is_election = True/False and election_full = True/False on the candidate_total api

**Comment by qqss88** [comment]
quick conclusions:
1. when election_full=false, those 2 endpoints will return different counts: one is candidate count, the other is aggregate total count per candidate per cycle
2. when election_full=true, those two endpoints tend to return the same count. The real count difference comes from data issues.

**Comment by pkfec** [comment]
@qqss88 can you summarize here on why this issue is blocked by #3700 ?

**Comment by qqss88** [comment]
Move this ticket to 'blocked' because some of the work by Jean on this ticket:
#3727
could help to resolve some issues discussed here. Will come back to redo some analysis when Jean's work is done. Thx

**Comment by qqss88** [comment]
after #3700 modification deployed, I repeated some analysis, the major difference is that /candidates/totals endpoints end up with much less items when election_full is set to False:

https://api.open.fec.gov/v1/candidates/totals/?election_full=true&api_key=DEMO_KEY&sort_null_only=false&sort_nulls_last=false&election_year=2020&office=P&page=1&per_page=20&sort_hide_null=false
count: 443

https://api.open.fec.gov/v1/candidates/totals/?election_full=false&api_key=DEMO_KEY&sort_null_only=false&sort_nulls_last=false&election_year=2020&office=P&page=1&per_page=20&sort_hide_null=false
count=137
